cmd/helm/registry_login.go

```diff
@@ -53,7 +53,7 @@ func newRegistryLoginCmd(cfg *action.Configuration, out io.Writer) *cobra.Comman
 		Short:             "login to a registry",
 		Long:              registryLoginDesc,
 		Args:              require.MinimumNArgs(1),
-		ValidArgsFunction: noCompletions,
+		ValidArgsFunction: cobra.NoFileCompletions,
```

Comment:
This turns off file completion but does not print the ActiveHelp message since an argument is expected, it is just that it should not be a file.  I'm plan to improve on this with a specific ActiveHelp message in a future PR.